launcher/container_runner.go

```diff
@@ -505,6 +508,16 @@ func (r *ContainerRunner) Run(ctx context.Context) error {
 		return fmt.Errorf("failed to fetch and write OIDC token: %v", err)
 	}
 
+	e := experiments.NewExperiments(r.logger)
```

Comment:
this callpoint should probably go somewhere in `main.go` and you should put the experiment flag settings in the LaunchSpec.